fisher.lua

```diff
@@ -377,8 +378,14 @@ function check_incoming_text(original, modified, original_mode, modified_mode, b
         message(3, 'recieved error text')
         if error_retry then
             error_retry = false
-            message(2, 'casting in %d seconds':format(settings.delay.cast))
-            windower.send_command('wait %d; lua i fisher cast':format(settings.delay.cast))
```

Comment:
You can use short circuit logic to modify these lines instead of having to use an if/else.
In the send_command, just replace:
`settings.delay.cast`
with:
`settings.delay.cast + (settings.random and math.random() or 0.0)`
Also, you forgot to use the float formatting in the string here/bellow.